frontend/src/components/Header.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const Header = (props) => {
+  const { pageTitle } = props;
+  return (
+    <div className="page-header">
+      <div className="cleanbc-banner">
+        <a
+          href="http://www.gov.bc.ca"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          <img src="../../BCID_H_rgb_rev.png" alt="Government of B.C." />
```

Comment:
You can import the file here to make sure paths are correct when it's built for prod: https://create-react-app.dev/docs/adding-images-fonts-and-files/